static/app/views/acceptProjectTransfer/index.tsx

```diff
@@ -42,8 +42,13 @@ class AcceptProjectTransfer extends DeprecatedAsyncView<Props, State> {
       },
       success: () => {
         const orgSlug = formData.organization;
-
-        this.props.router.push(normalizeUrl(`/organizations/${orgSlug}/projects/`));
+        const projectSlug = this.state?.transferDetails?.project.slug;
+        const sentryUrl = ConfigStore.get('links').sentryUrl;
+        if (projectSlug === null) {
```

Comment:
@ceorourke I think it would be better to do something like `!projectSlug` in case it's undefined